packages/terra-framework-docs/src/terra-dev-site/test/tabs/Tabs/ControlledTabs.test.jsx

```diff
@@ -59,6 +88,9 @@ class ControlledTabs extends React.Component {
           {tab1}
           {tab2}
           {tab3}
+          {tab4}
+          {tab5}
+          {tab6}
```

Comment:
Can we include tab with icon also here to validate them on WDIO for alignment and other style properties